app/furniture/marketplace/routes.rb

```diff
@@ -15,6 +15,7 @@ def self.append_routes(router)
         router.resource :stripe_account, only: [:show, :new, :create]
         router.resources :stripe_events
         router.resources :tax_rates
+        router.resources :payment_settings
```

Comment:
❓ Should I do `only: [:index]` or it doesn't really matter?

Reply:
`only: [:index]` is a bit tighter; but it's a six-in-one-half-dozen-in-the-other kind of thing.